.coveragerc

```diff
@@ -1,6 +1,13 @@
 [report]
+# Regexes for lines to exclude from consideration
+exclude_lines =
+    # Don't complain if tests don't hit defensive assertion code:
+    raise ImportError
+    raise NotImplementedError
+
 fail_under = 84
```

Comment:
I would propose to raise this to 85 then.

Reply:
Updated.